src/components/Text/Text.tsx

```diff
@@ -33,28 +33,33 @@ interface TextProps
     inverted?: boolean;
     /**
      * Adjust color to indicate secondary information
+     * @deprecated
      */
     weak?: boolean;
+    /**
+     * Adjust color to indicate secondary information
+     */
+    secondary?: boolean;
     /**
      * Adjust color to display a disabled text element
      */
     disabled?: boolean;
 }
 
-function determineTextColor({ weak, inverted, disabled }: TextProps) {
+function determineTextColor({ weak, secondary, inverted, disabled }: TextProps) {
     if (disabled) {
-        return inverted ? Colors.AUTHENTIC_BLUE_550 : Colors.AUTHENTIC_BLUE_350;
+        return inverted ? SemanticColors.text.disabledInverted : SemanticColors.text.disabled;
```

Comment:
Please use the theme get method, so we have a single place to update the colors (with the theme). In addition this makes it possible for users to consistently override a single color across the components (see here https://github.com/freenowtech/wave/blob/main/src/components/Button/Button.tsx#L23).

```suggestion
        return inverted ? get('semanticColors.text.disabledInverted')(props) : get('semanticColors.text.disabled')(props);
```

Be sure to also pass all the props to the function, instead of descructuring them in L49. This way, the required props (theme for example) can be passed correctly to the get function.